tests/api/loans/test_loans_rest.py

```
@@ -658,3 +658,97 @@ def test_librarian_request_on_blocked_user(
     assert res.status_code == 403
     data = get_json(res)
     assert "blocked" in data.get("message")
+
+
+def test_keep_history_loan_visibility(
+    client,
+    librarian_martigny,
+    loc_public_martigny,
+    item_on_loan_martigny_patron_and_loan_on_loan,
+    circulation_policies,
```

Ruff (0.15.2)

[warning] Unused function argument: `circulation_policies`

(ARG001)